index.d.ts

```diff
@@ -2976,7 +2976,7 @@ declare module sequelize {
     [string | col | fn | literal, string] |
     [Model<any, any> | { model: Model<any, any>, as: string }, string, string] |
     [Model<any, any>, Model<any, any>, string, string];
-  export type Order = OrderItem | OrderItem[];
+  export type Order = string | fn | col | literal | OrderItem[];
```

Comment:
I think your intention was to add `string[]` to `OrderItem` rather than removing it here. Tests are failing in travis ci.